tests/Browser/2-critical.testcafe.js

```diff
@@ -42,7 +42,7 @@ test("Applicant Profile - My Skills", async t => {
     .expect(Selector("h3").withText("Writing my application").visible)
     .ok()
     .click(Selector("button").withText("Got it!"))
-    .wait(300)
+    .wait(1234)
```

Comment:
I'd try changing it to a smaller value first, to see if it fixes the issue (500 maybe). And change the upstream to the PR with the test failing (#2499).

Reply:
I just made a new PR off the latest dev #2514 which was breaking as well. So my thought was to make the change to the dev branch. But yes I can try making the value smaller.

Reply:
Looks like lower time works 👍